src/components/concentration-display/Concentration.tsx

```diff
@@ -34,6 +35,10 @@ const Concentration: React.FC<AgentProps> = ({
     const { isPlaying, maxConcentration, page, getAgentColor } =
         useContext(SimulariumContext);
     const [width, setWidth] = useState<number>(0);
+
+    const MARGINS = 64.2;
```

Comment:
Nit: move to constants file?

Reply:
it's only being used by this file, so I think it's better to keep it here